examples/rust_with_cffi/setup.py

```diff
@@ -19,11 +19,7 @@
     ],
     packages=["rust_with_cffi"],
     rust_extensions=[
-        RustExtension(
-            "rust_with_cffi.rust",
-            py_limited_api=True,
-            features=[] if platform.python_implementation() == 'PyPy' else ["pyo3/abi3"]
-        ),
+        RustExtension("rust_with_cffi.rust"),
```

Comment:
I think we should use auto here.

Reply:
👍 sounds reasonable. `"auto"` is the default, but can't hurt for us to be explicit to help users discover it.